doc/nrf/protocols/bt/bt_qualification/index.rst

```diff
@@ -0,0 +1,124 @@
+* Your new product can be added as a new entry to existing list of product entries as there was no design change in the new product.
+* You want to switch to another major |NCS| version for an existing product.
+* In some cases, if you want to switch to another minor |NCS| version for an existing product.
+
+It may happen that you would need to alter the configuration of the qualified design or slightly modify small parts of the existing design to better fit your application needs.
+If you make such changes, you must check the following:
+
+* Ensure that the resulting design does not add new features (ICS entries) that are not present in Nordic Semiconductor's qualified design.
+* Ensure that the resulting design does not break the qualification of the existing design.
+
+You can do so by running relevant qualification tests using test tools.
+
+Switching to another SDK version
+--------------------------------
+
+If you want to switch to another |NCS| version for an existing product, complete the following steps:
+
+1. Go to the `Nordic Semiconductor Infocenter`_ and search for the SoC or SiP model you use.
+
+#. Navigate to the relevant Compatibility Matrix directory.
+
+#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:
+
+   * That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.
+   * That relevant Host and SoftDevice Controller Subsystems implemented in the |NCS| version you want to use are pre-qualified.
+     If yes, check the following:
+
+     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones currently used for your product, then no further action is required.
+     * If the version you want to use has at least one QDID number that is different from the numbers already used for your product, then a new qualification is required.
+       Log into the `Launch Studio`_ tool and follow the steps for `Qualification process with no required testing`_ option.
+
+Qualification listing
+*********************
+
+The `Bluetooth Qualification Listing`_ search lists all qualified designs.
+Use it in the following cases:
+
+* To check if your qualification process succeeded.
+* To search for existing designs that can be used in your product.
+* To check if you can add your device model to an existing qualification entry.
+
+Matter certification by inheritance
+***********************************
+
+Bluetooth QDIDs may be further used to obtain the Matter certification by inheritence.
+See the :ref:`ug_matter_device_certification_reqs_dependent` section for details.
+
+Support
+*******
+
+For details about the qualification process, see the `Qualifications and listings`_ page, or ask `Bluetooth Qualification Consultants`_ for advice.
+
+In case of any questions regarding Nordic Semicondutor's qualified designs and their use in your products, contact `DevZone`_.
```

Comment:
```suggestion
In case of any questions regarding Nordic Semicondutor's qualified designs and their use in your products, contact Nordic Semiconductor's technical support on `DevZone`_.
```